airqo-platform/AirQo-frontend
# Join locate

- What does this PR do?
This PR has a landing page with the option to join the Locate Product
- What JIRA task is related to this PR?
The links to the JIRA boards task are:
1. [Planning Space](https://airqoteam.atlassian.net/browse/LOC-7).
2. [Select Locations For Planning Space](https://airqoteam.atlassian.net/browse/LOC-9)
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/locate` Then run `npm install` to get all the necessary dependencies Afterwards run `npm start` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
No end points for testing
